docs/codeql/codeql-cli/using-the-codeql-cli/creating-codeql-query-suites.rst

```diff
@@ -150,6 +150,15 @@ use::
        - high
        - very-high
 
+To create a suite that selects all queries with ``@kind problem`` and an
+``@tag`` starting with ``external/cwe`` from the ``codeql-javascript`` QL pack,
+use::
+
+   - qlpack: codeql-javascript
+   - include:
+       kind: problem
+       tags contain: /^external\/cwe\//
```

Comment:
```suggestion
       tags contain: /^external\/cwe/
```
Either there shouldn't be a trailing escaped slash here, or it should be mentioned in the description.